### PR TITLE
Feeds Tests. 

### PR DESCRIPTION
**Pull request by RickKwok**
All passed tests. Fixed issues.

### CODE REVIEW

**Comment by jchristi** [comment]
Merge conflicts. Wait until Zach's PR (#7) for FeedCategory is merged before resolving.

**Comment by jchristi** [comment]
Just merged #7, go ahead and rebase or re-merge your branch and re-submit.

**Comment by RickKwok** [comment]
Hi @jchristi Jeremy, I have just removed all the conflicts. Would you check at your convenience please?

File: models/FeedCategory.js
Comment (jchristi) [inline models/FeedCategory.js]: We got rid of this method, it may have been the cause of performance issues in the old branch. Associations are performed in `models/index.js` now.

File: tests/models/Feed_test.js
Comment (jchristi) [inline tests/models/Feed_test.js]: This function has been moved to `tests/index.js` so every model test can make use of it.

**Review by jchristi — CHANGES_REQUESTED** [review]
There are a number of ESLint errors, see the [Travis log](https://travis-ci.org/jchristi/syndicat/jobs/218520777) for details. You can run a lint locally with the `make lint` command.

**Pull request closed without merging**
